app/ducks/walletActions.js

```diff
@@ -101,6 +105,7 @@ export const fetchWallet = () => async (dispatch, getState) => {
     balance: accountInfo.balance,
     changeDepth: accountInfo.changeDepth,
     receiveDepth: accountInfo.receiveDepth,
+    type: accountInfo.type,
```

Comment:
I actually had a TODO here in my branch: 

```js
// TODO: remove all the above crap and just pass account object
```

It seems tedious to add and verify each inidivual little property of `accountInfo` -- lets just add the whole damn object to the wallet state. Eventually we can also refactor out `address`, `balance`, `changeDepth` etc... But by chucking the whole object in there we get `type` (for "multisig") and also `accountKey` which will be how we can display our own xpub on the multisig dashboard